orderer/common/localconfig/config_test.go

```diff
@@ -314,3 +314,17 @@ func TestChannelParticipationDefaults(t *testing.T) {
 	require.Equal(t, cfg.ChannelParticipation.Enabled, Defaults.ChannelParticipation.Enabled)
 	require.Equal(t, cfg.ChannelParticipation.MaxRequestBodySize, Defaults.ChannelParticipation.MaxRequestBodySize)
 }
+
+func TestKeepaliveDefaults(t *testing.T) {
+	os.Setenv("ORDERER_GENERAL_KEEPALIVE_CLIENTINTERVAL", "30s")
+	defer os.Unsetenv("ORDERER_GENERAL_KEEPALIVE_CLIENTINTERVAL")
+	cleanup := configtest.SetDevFabricConfigPath(t)
+	defer cleanup()
+
+	cc := &configCache{}
+	cfg, err := cc.load()
+	require.NotNil(t, cfg, "Could not load config")
+	require.NoError(t, err, "Load good config returned unexpected error")
```

Comment:
```suggestion
	require.NoError(t, err, "Load good config returned unexpected error")
	require.NotNil(t, cfg, "Could not load config")
```

a better practice would be to first check for the error and only then for config not being nil.

Reply:
sure. i will update the PR.